cypress/e2e/invoices/cancelling-approved-invoices.cy.js

```diff
@@ -55,7 +55,7 @@ describe('ui-invoices: Cancelling approved invoices', () => {
 
               cy.visit(TopMenu.fundPath);
               Helper.searchByName(defaultFund.name);
-              Helper.selectFromResultsList();
+              Funds.selectFund(defaultFund.name);
```

Comment:
![image](https://user-images.githubusercontent.com/99860864/208080771-eee7f914-b76a-4338-a663-68e575144e7a.png)
problems with eslint and you imported same module two times with different names (Helper SearchHelper) please delete one